src/bitdrift_public/protobuf/filter/v1/filter.proto

```diff
@@ -62,10 +51,50 @@ message Filter {
         MATCHING_ONLY = 2;
       }
 
+      message SetFieldValue {
+        message ExistingField {
+          // The name of an existing field.
+          string name = 1 [(validate.rules).string = {min_len: 1}];
+        }
+
+        oneof value {
+          option (validate.required) = true;
+
+          // The constant string value.
+          string string_value = 1;
+          // The value of an existing field.
+          ExistingField existing_field = 2;
+        }
+      }
+
       string name = 1 [(validate.rules).string = {min_len: 1}];
-      string value = 2 [(validate.rules).string = {min_len: 1}];
+      // The value to set the field to.
+      SetFieldValue value = 2 [(validate.rules).message.required = true];
+      // The type of the field to set.
       FieldType field_type = 3 [(validate.rules).enum = {defined_only: true not_in: 0}];
+      // Whether a transform is allowed to override an existing field. If no and the field already exists,
+      // the transform is a no-op.
+      bool allow_override = 4;
     }
+
+    // Removes a specified field.
+    message RemoveField {
+      // The name of the field to remove.
+      string name = 1 [(validate.rules).string = {min_len: 1}];
+    }
+  }
+
+  // Scrubs the content of a given field, replaces substring(s) matching a given regex with a provided
+  // substitution string.
+  message ScrubField {
+    // The name of the field to scrub. If the field doesn't exist, nothing happens.
+    string name = 1 [(validate.rules).string = {min_len: 1}];
+    // The regex used to find portions of a string that should be replaced. Capture groups can be used to
+    // extract portions of the input string and regerence them later in a `substition` string.
+    string pattern = 2 [(validate.rules).string = {min_len: 1}];
+    // The string that matching portion of the subject string should be replated with. Allows for referencing
+    // capture groups from the `pattern` regex.
```

Comment:
```suggestion
    // The string that matching portions of the subject string should be replaced with. Allows for referencing
    // capture groups from the `pattern` regex.
```

Probably worth explicitly documenting that we are using Rust regex syntax also for everything in here.